source/stable-release_iot/feature-delta-iot.rst

```diff
@@ -0,0 +1,76 @@
+.. _feature-delta-iot:
+
+Feature Delta Offered by Celadon IoT
+####################################
+
+Following are the additional features provided by Celadon IoT.
+
+.. contents::
+    :depth: 1
+    :local:
+* 100% Google Conformance on Comet Lake(CML),Tiger Lake(TGL) & Elkhart Lake(EHL) with Production ready
+* Supporting Comet Lake(CML),Tiger Lake(TGL) & Elkhart Lake(EHL) on single baseline and Single Image
+* Elkhart Lake(EHL) Platform supported on Celadon with 100% compliance
```

Comment:
Why are we calling out EHL again, we covered 100% conformance and EHL in above two points already. Do we have a conclusion on the public platform that can be used along with this release?

Reply:
Updated as per the Suggestion : Removed the Bulletin